app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/filtersControl/filtersControl.jsx

```diff
@@ -59,7 +66,7 @@ const messages = defineMessages({
   (state) => ({
     userId: userIdSelector(state),
     activeProject: activeProjectSelector(state),
-    activeFilterId: (selector(state, 'filterIds') && selector(state, 'filterIds')[0]) || '',
+    activeFilterId: activeFilterIdSelector(state),
```

Comment:
I've just thought: we use `FilterControls` inside of a `FieldProvider`, so we should receive `filterIds` as the `value` prop.
Can `value` prop be used instead of this selector? For example, introduce an instance method `getActiveFilterId`.

Reply:
Oh! Good idea! Thank's for your suggestion.